packages/react-components/react-button/src/stories/MenuButton/MenuButtonIcon.stories.tsx

```diff
@@ -6,10 +6,16 @@ import {
   FilterFilled,
   FilterRegular,
 } from '@fluentui/react-icons';
-import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { makeStyles } from '@griffel/react';
-import { MenuButton } from '@fluentui/react-button';
+import {
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  Tooltip,
+  makeStyles,
+  MenuButton,
```

Comment:
_nit:_ Order these alphabetically

```suggestion
  makeStyles,
  Menu,
  MenuButton,
  MenuItem,
  MenuList,
  MenuPopover,
  MenuTrigger,
  Tooltip,
```